### PR TITLE
#531: Add Node.JS + Express integration documentation

### LINKED ISSUE #531: Create a Node + Express page under Integrations
From PR #502 by @latvaaho:

> This change [below] was needed when taking a vanilla node.js with express project template to get form posting to work properly out-of-box. As I an not that experience front-end developer, it took a while to find out why I wasn't getting the editor's content into the body of submitted form. Finally the only issue was to have also name in addition to id defined for the textarea, and after that everything worked as a breeze.

```
<body>
  <h1>TinyMCE Quick Start Guide</h1>
    <form method="post">
 -    <textarea id="mytextarea">Hello, World!</textarea>
 +    <textarea id="mytextarea" name="mytextarea">Hello, World!</textarea>
    </form>
  </body>
  </html>
```

We should consider creating a Node + Express integration doc.

cc/ @Afraithe

### PR DESCRIPTION
**Pull request by tylerkelly13**
Related Ticket: https://github.com/tinymce/tinymce-docs/issues/531

Description of Changes:
* Add a walkthrough for creating a Node.js + Express app containing TinyMCE
* resolves #531

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: _includes/integrations/expressjs-quick-start.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    var usersRouter = require('./routes/users');
+
+    var app = express();
+
+    app.use(logger('dev'));
+    app.use(express.json());
+    app.use(express.urlencoded({ extended: false }));
+    app.use(cookieParser());
+    app.use(express.static(path.join(__dirname, 'public')));
+    /* New Route to the {{site.productname}} Node module */
```
Comment (lnewson) [inline _includes/integrations/expressjs-quick-start.md:61]: Maybe add a new line before this to highlight it's the new part?

File: _includes/integrations/expressjs-quick-start.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+
+1. Download and install the {{site.productname}} and Express JS Node modules (and their dependencies):
+
+    ```sh
+    $ npm install
+    ```
+
+1. Using a text editor, open `/path/to/my_{{site.prodnamecode}}_app/app.js` and add the following route:
+
+    ```jsx
```
Comment (lnewson) [inline _includes/integrations/expressjs-quick-start.md:39]: This shouldn't be `jsx`, it's just plain `js`.

File: _includes/integrations/expressjs-quick-start.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+    app.use('/', indexRouter);
+    app.use('/users', usersRouter);
+
+    module.exports = app;
+    ```
+
+1. Create a new javascript file containing the {{site.productname}} configuration (`tinymce.init`), such as:
+
+    ```sh
+    $ touch public/javascripts/my-{{site.prodnamecode}}-config.js
```
Comment (lnewson) [inline _includes/integrations/expressjs-quick-start.md:73]: I'm pretty sure `touch` doesn't work on Windows. Does that matter?

Comment (tylerkelly13) [reply _includes/integrations/expressjs-quick-start.md:73]: need to mention on windows that they use `type nul >public/javascripts/my-{{site.prodnamecode}}-config.js`

File: _includes/integrations/expressjs-quick-start.md (hunk 1 above)
Comment (tylerkelly13) [reply _includes/integrations/expressjs-quick-start.md:62]: Fixed by https://github.com/tinymce/tinymce-docs/pull/1625/commits/1f077fff34ce2fbd9db25ca8b5bfee299e9e881c

File: _includes/integrations/expressjs-quick-start.md (hunk 2 above)
Comment (tylerkelly13) [reply _includes/integrations/expressjs-quick-start.md]: Fixed by https://github.com/tinymce/tinymce-docs/pull/1625/commits/1f077fff34ce2fbd9db25ca8b5bfee299e9e881c

File: _includes/integrations/expressjs-quick-start.md (hunk 3 above)
Comment (tylerkelly13) [reply _includes/integrations/expressjs-quick-start.md:76]: Fixed by https://github.com/tinymce/tinymce-docs/pull/1625/commits/1f077fff34ce2fbd9db25ca8b5bfee299e9e881c

**Pull request merged**